src/group_impl.h

```diff
@@ -326,8 +324,6 @@ static void secp256k1_gej_double_var(secp256k1_gej *r, const secp256k1_gej *a, s
 
     if (rzr != NULL) {
         *rzr = a->y;
-        secp256k1_fe_normalize_weak(rzr);
```

Comment:
I haven't looked at the callers, but removing this normalize_weak is a little sketchy.

Reply:
My belief is that this normalize_weak call only exists because of the multiply by 2 on the next line, which would otherwise operate on an input of unknown magnitude.

Reply:
Maybe it's time to work on https://github.com/bitcoin-core/secp256k1/issues/1001 .

Reply:
I doubt there is a problem here, but performance impact is negligible and I don't want to be losing sleep, so I've restored the _normalize_weak. I think it can be cleaned up after/with #1001 and/or #1032.